cms/templates/js/show-correctness-editor.underscore

```diff
@@ -0,0 +1,48 @@
+<form>
+<h3 class="modal-section-title" id="show_correctness_label"><%- gettext('Correctness Visibility') %></h3>
+<div class="modal-section-content show-correctness">
+    <ul class="list-fields list-input" role="group" aria-labelledby="show_correctness_label">
+        <li class="field-radio">
+            <label class="label">
+                <input class="input input-radio" name="show-correctness" type="radio" value="always" aria-described-by="always_show_correctness_description">
+                <%- gettext('Always show correctness') %>
+            </label>
+            <p class='field-message' id='always_show_correctness_description'>
+                <%- gettext('When learners submit an answer to a problem, they can immediately see whether the answer is correct or incorrect, and the score received.') %>
+            </p>
+        </li>
+        <li class="field-radio">
+            <label class="label">
+                <input class="input input-radio" name="show-correctness" type="radio" value="never" aria-described-by="never_show_correctness_description">
+                <%- gettext('Never show correctness') %>
+            </label>
+            <p class='field-message' id='never_show_correctness_description'>
+                <%- gettext('Learners are never allowed to see whether their answers to problems are correct or incorrect, or the score received.') %>
+            </p>
+        </li>
+        <li class="field-radio">
+            <label class="label">
+                <input class="input input-radio" name="show-correctness" type="radio" value="past_due" aria-described-by="show_correctness_past_due_description">
+                <%- gettext('Show correctness when subsection is past due') %>
+            </label>
+            <p class='field-message' id='show_correctness_past_due_description'>
+                <% if (self_paced) { %>
+                    <%- gettext('Learners are not allowed to see whether their answers to problems were correct or incorrect, or the score received, until after the Course End Date has passed.') %>
+                <% } else { %>
+                    <%- gettext('Learners are not allowed to see whether their answers to problems were correct or incorrect, or the score received, until after the Grading Due Date has passed.') %>
+                <% } %>
+            </p>
+        </li>
+    </ul>
+
+    <% if (self_paced) { %>
```

Comment:
I'm not sure why I was trying to be so fancy here..  Have just moved the warning text into the main paragraph, and am now showing it all the time, so screen-readers and people-readers can always see it.  This simplifies the code and tests too.  cf https://github.com/edx/edx-platform/pull/14737/commits/541b3c9d4d39e3d84910ea242c86c9dc4569e069#diff-f60a9f957169beadefa5fedf9607dfeaR27.